### DISCUSSION TITLE
way to declare and install a k8s operator using helmfile

### DISCUSSION THREAD

**Discussion by sibelius** [helmfile/helmfile#1345] [Ideas]
most operators use kubectl apply -f, like tekton, to install k8s operators

https://tekton.dev/docs/installation/pipelines/

```
kubectl apply --filename https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml
```

how to declare this in a helm chart or helmfile ?

**Comment by yxxhero** [comment]
@sibelius first of all. thanks for using helmfile. you can try put this into a dir. and set the chart of release to the dir path.
